com.github.rajsolai.response.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.rajsolai.response
+
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+base: io.elementary.BaseApp
+base-version: 'juno-20.08'
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.response
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--socket=x11'
+  - '--filesystem=home'
+  - '--share=network'
+  - '--device=dri'
+
+modules:
+  - name: response
+    buildsystem: meson
+    sources:
+      - type: archive
+        path: https://github.com/RajSolai/Response/archive/refs/tags/1.0.0.tar.gz
```

Comment:
1. I think you want `url` here
2. You'll need the `sha256` checksum added.

Reply:
Thanks , Changing now !